modules/cco-determine-mode-cli.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-cli.adoc
+// * authentication/managing_cloud_provider_credentials/about-cloud-credential-operator.adoc
+
+:_content-type: PROCEDURE
+
+ifeval::["{context}" == "updating-cluster-cli"]
+:cli:
+endif::[]
+ifeval::["{context}" == "about-cloud-credential-operator"]
+:about-cco:
+endif::[]
+
+[id="cco-determine-mode-cli_{context}"]
+= Determining the Cloud Credential Operator mode by using the CLI
+
+You can determine what mode the Cloud Credential Operator (CCO) is configured to use by using the CLI.
+
+[NOTE]
+====
+Only Amazon Web Services (AWS), global Microsoft Azure, and Google Cloud Platform (GCP) clusters support multiple CCO modes.
+====
+
+.Prerequisites
+
+* You have access to an {product-title} account with cluster administrator permissions.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Log in to `oc` on the cluster as a user with the `cluster-admin` role.
+
+. To determine the mode that the CCO is configured to use, enter the following command:
++
+[source,terminal]
+----
+$ oc get cloudcredentials cluster \
+  -o=jsonpath={.spec.credentialsMode}
+----
++
+The following output values are possible, though not all are supported on all platforms:
++
+--
+* `''`: The CCO is operating in the default mode. In this configuration, the CCO operates in mint or passthrough mode, depending on the credentials provided during installation.
```

Comment:
For the non-upgrade case, I want to add a step here to show how to look at the annotation and determine which it is.